NDcPP_v2_2e.adoc

```diff
@@ -907,13 +908,19 @@ _For pNDs, ‘on the TOE itself’ or ‘locally’ means on storage inside or d
 
 _For vNDs, local storage is any storage accessible by TOE software. In a virtualized environment, ‘local’ storage is under the control of the VS and may be physically located on the local host, but it could also be located on a network drive or storage array._
 
-*FAU_STG_EXT.1.3* The TSF shall [selection: _drop new audit data, overwrite previous audit records according to the following rule: [assignment: rule for overwriting previous audit records], [assignment: other action]_] when the local storage space for audit data is full.
+===== FAU_ STG_EXT.1/LocSpace Local Audit Event Storage
+
+*FAU_STG_EXT.1/LocSpace Local Audit Event Storage*
+
+*FAU_STG_EXT.1.1/LocSpace* The TSF shall maintain a [_selection: log file, database, buffer, assignment:[other local logging method]_] of audit records in the event that an interruption of communication with the remote syslog server occurs.
+
+*FAU_STG_EXT.1.2/LocSpace* The TSF shall be able to store [_selection: persistent, non-persistent_] audit records locally with a minimum storage size of [_assignment: number of records, file/buffer size(s)_].
```

Comment:
Would it be better if the SFR and associated Assurance Activities used the terms " non-volatile" and "volatile" storage?  

For example:  The TSF shall be able to store audit records locally in [_selection: non-volatile, volatile_] storage with a minimum storage size of [_assignment: number of records, file/buffer size(s)_]

Reply:
I do not think we should change this. I believe the use of persistent vs. volatile is appropriate when it comes to discussing logging.